### DISCUSSION TITLE
mapping key in configs isn't working.

### DISCUSSION THREAD

**Discussion by hyper-dot** [nvim-neo-tree/neo-tree.nvim#949] [General]
I tried to map l and h to open and close folders but its not working. Is it because it is already mapped ??

Here's my config :

```lua
neotree.setup({
	git_status = {
		symbols = {
			untracked = "",
			ignored = "",
			unstaged = "",
			staged = "",
			conflict = "",
		},
	},
	window = {
		position = "left",
		width = 30,
	},
	mappings = {
		"l" == "open",
	},
	filesystem = {
		filtered_items = {
			hide_dotfiles = false,
			hide_gitignored = false,
			hide_by_pattern = {
				"*/node_modules/*",
			},
		},
	},
})

```

**Comment by nhat-vo** [comment]
First, the syntax in lua in defining such map is either `['l'] = 'open'` or `l = 'open'`. Funny how your neovim doesn't throw a syntax error.

Second, the `mappings` field should be defined inside the `window` field.

**Reply by hyper-dot** [reply]
Thank you. Fixed it !!

**Reply by technbuzz** [reply]
I also made the same mistake, the config is so big that I lost tracking the opening brackets. I think the configs aren't well designed. May be not well explained. We might need better docs or explainer video
